requirements/docs.txt

```diff
@@ -1,4 +1,4 @@
-sphinx>=3.0, !=3.5  # fails with sphinx.ext.viewcode
+sphinx>=3.0, !=3.5, <4.0  # fails with sphinx.ext.viewcode  # fails with sphinx_paramlinks with 4.0.0
```

Comment:
There was no other release between 3.5 and 4.0

Reply:
I was wondering why it was not using `!=3.5` in the very last successful runs even though we specified `!=3.5`:
```
Sphinx                        3.5.4
```
https://github.com/PyTorchLightning/pytorch-lightning/runs/2535513652